src/templates/modal.js

```diff
@@ -21,14 +21,19 @@ class ModalHeader extends Component {
 
     render() {
         const {
-            children, closeButton, inverse, title, titleTruncate,
+            children,
+            closeButton,
+            inverse,
+            title,
+            titleTruncate,
+            style,
```

Comment:
Nitpick: Let's please go ahead and keep these in alphabetical order.  So let's move `style` up above `title`.

Thank you for moving them to separate lines per our preferred style/convention.

Reply:
done

Reply:
Thank you so much @raoufr